interp/src/macros.rs

```diff
@@ -17,6 +17,7 @@
 /// `StdAdd::new(bindings: ir::Params)` and `StdAdd::from_constants(ports)`
 ///
 /// TODO(rachit): $out_width is never used.
+/// (TODO: Griffin) fix the copy-paste
```

Comment:
Does this need to be addressed before merge?

Reply:
No, it's just gross and should probably be better in the future.